src/lib/netlist/devices/nld_roms.cpp

```diff
@@ -96,6 +96,59 @@ namespace netlist
 			nld_power_pins m_power_pins;
 		};
 
+		NETLIB_OBJECT(mcm14524_rom)
+		{
+			NETLIB_CONSTRUCTOR(mcm14524_rom)
```

Comment:
This is a CMOS device - use NETLIB_CONSTRUCTOR_MODEL(mcm14524_rom, "CD4XXX")

Reply:
Fixed.